package.json

```diff
@@ -7,35 +7,36 @@
     "node": ">=20.0.0"
   },
   "scripts": {
-    "test": "turbo run test",
-    "test:coverage": "turbo run test:coverage",
-    "test:perf": "turbo run test:perf",
-    "test:quality": "turbo run test:quality",
-    "test:packages": "turbo run test --filter=./packages/*",
-    "test:coverage:packages": "turbo run test:coverage --filter=./packages/*",
-    "test:coverage:report": "node scripts/merge-coverage.js",
-    "test:integration": "turbo run test:integration",
+    "test": "wireit",
+    "test:coverage": "wireit",
+    "test:perf": "wireit",
+    "test:quality": "wireit",
+    "test:packages": "npm run test --workspaces --if-present",
```

Comment:
isn't that what `test` does?

Reply:
Yes it does, context tells me that it should be invoking a test that exercises the pacakges, i.e. the vsix. 

Let me see what I can find. I will nuke if nothing relevant exists today.